patch-diff-report-tool/src/main/java/com/github/checkstyle/data/DiffReport.java

```diff
@@ -147,10 +147,17 @@ public void getDiffStatistics() {
                 : records.entrySet()) {
             final List<CheckstyleRecord> list = entry.getValue();
             for (CheckstyleRecord rec : list) {
-                statistics.addSeverityRecord(rec.getSeverity(),
-                        CheckstyleReportsParser.DIFF_REPORT_INDEX);
-                statistics.addModuleRecord(rec.getSource(),
-                        CheckstyleReportsParser.DIFF_REPORT_INDEX);
+                if (rec.getIndex() == CheckstyleReportsParser.BASE_REPORT_INDEX) {
+                    statistics.addSeverityRecordRemoved(rec.getSeverity());
+                    statistics.addModuleRecordRemoved(rec.getSource());
+                }
+                else if (rec.getIndex() == CheckstyleReportsParser.PATCH_REPORT_INDEX) {
+                    statistics.addSeverityRecordAdded(rec.getSeverity());
+                    statistics.addModuleRecordAdded(rec.getSource());
+                }
+                else {
+                    throw new AssertionError();
```

Comment:
We should remove all these assertions as we are trying to get higher code coverage and there are no new tests to hit these.

Reply:
done